mobile/src/components/Card.tsx

```diff
@@ -26,4 +26,4 @@ function Card({
   );
 }
 
-export default styled(Card);
+export default memo(styled(Card));
```

Comment:
Was there any reason for this? I think this might be an unnecessary use of memo given how few elements Card contains

Reply:
It seemed to prevent re-renders. That's why I have it there, is there a better way to prevent re-renders?